nautobot/extras/views.py

```diff
@@ -729,6 +729,67 @@ class DynamicGroupUIViewSet(NautobotUIViewSet):
     table_class = tables.DynamicGroupTable
     action_buttons = ("add",)
 
+    class FilterBaseTextPanel(object_detail.BaseTextPanel):
+        def get_value(self, context):
+            obj = get_obj_from_context(context)
+            if obj.group_type == "dynamic-filter":
```

Comment:
Can we replace the literal strings here and below with `DynamicGroupTypeChoices` values, e.g.:

```suggestion
            if obj.group_type == DynamicGroupTypeChoices.TYPE_DYNAMIC_FILTER:
```